README.md

```diff
@@ -1,2 +1,213 @@
+> commitment, not only to the Project, but to the stability and vitality of
+> the Community as a whole.
+
+Node.js does not exist within a vacuum. It thrives because of the ecosystem that
+it has enabled, and that has grown and thrived around it. Our responsibility is
+to the entire community of JavaScript developers.
+
+> It is vital that users trust the Contributors to have their best interests in
+> mind while making decisions affecting the overall direction of the platform.
+> This means having defined guidelines, stable APIs, a clear roadmap, and a
+> predictable release schedule. The user community and developer ecosystem
+> must be included in the decision making process.
+
+All decisions, including those made by this Committee must be open for
+discussion, debate, and consideration by the members of the community directly
+impacted. This includes the decision of whether to advance forward with the
+merger of the Foundations. This community values not only radical transparency,
+but radical collaboration, at every level.
+
+> Adoption of new capabilities and features within Node.js must be carefully
+> balanced by the expressed needs of the users and the community. Change just
+> for the sake of change must be avoided and a guaranteed long term support
+> policy must be established.
+
+Avoiding change for the sake of change applies as much to the governance and
+structure of the Foundation as it does to the technical specifics of the various
+projects. If we are to decide that changing the Foundation structure and merging
+the two organizations is something that is in the best interests of the
+community, we must be capable of articulating exactly what the benefits are
+and who those benefits serve.
+
+> Node.js owes much of its success to the existing ecosystem of module and
+> add-on developers. The Node.js project should do everything it can to support
+> the continued growth and stability of that ecosystem. This includes, but is
+> not limited to, keeping Node.js focused on providing a minimal kernel of core
+> functionality while deferring, as much as possible, additional capabilities
+> and features to add-on modules or applications.
+
+As above, the "small core" philosophy applies as much to the Governance model
+as it does to the code. At no point should the Foundation and the project
+leadership assume that "it knows best" when it comes to understanding what is
+best for the community as a whole.
+
+> In order to best serve the Node.js Community as a whole, the development,
+> release and issue management processes adopted by the Node.js Foundation
+> TSC must reflect these guiding principles.
+
+Likewise, should the merger proceed, the structure and governance of the new
+Foundation must first and foremost reflect the needs of the community,
+addressing real problems faced by that community.
+
+## What Problems Are We Trying To Solve?
+
+TBD
+
+## What Additional Goals Do We Have?
+
+TBD
+
+## What Concerns Have Been Raised?
```

Comment:
Can this be transformed from concerns to goals?

for example

"Lack of Transparency in process" could be changed to "We aim to have a transparent process"

This may not accomplish what this section is attempting to document, but I am concerned that this section comes across fairly negative, and based on perception rather than reality. Broad comments like "Multiple technical project members" can be read in very different ways, and does not reflect that other project members may not feel that way. 

please take a look at the identified concerns in the [public town hall deck](https://docs.google.com/presentation/d/e/2PACX-1vQwmJxbKAU0ARz03R7viAcawoYL0pATNbNxm1p1Z0R3efIVHjSqYiBdSTAfxXIMyocRGxGE_qKFFaJ6/pub?start=false&loop=false&delayms=3000&slide=id.g437c366200_6_0) as an example of how concerns have currently been framed. I'm more than happy to translate those concerns to a section like we see below, including copy about each concern, if that would be helpful

Reply:
I'd prefer to keep these as a separate list of explicitly acknowledged concerns, then use those to derive a list of goals specifically targeted at addressing those concerns

Reply:
Specifically, not all of these points *can* or *should* be transformed into goals and in so doing we actually transform the intent and meaning of those who have raised those concerns. For instance, the concern about "*Disagreement with stated goals*" speaks specifically in opposition of the move for a merged Foundation. It would be disingenuous for us to transform that into a goal *for* a merged Foundation.